### DISCUSSION TITLE
issue during installing requirement.txt

### DISCUSSION THREAD

**Discussion by GeneralSubhra** [metavoiceio/metavoice-src#45] [General]
subprocess-exited-with-error

 × python setup.py egg_info did not run successfully.
 │ exit code: 1
 ╰─> [6 lines of output]
   Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "C:\Users\Subhranil\AppData\Local\Temp\pip-install-_ng7_t4n\flash-attn_bddf0d5e226d4d5395a7f4bfd047b983\setup.py", line 9, in <module>
     from packaging.version import parse, Version
   ModuleNotFoundError: No module named 'packaging'
   [end of output]

 note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

I am getting this isuue . How to solve this?python version Im using is 3.10

**Comment by niie2333** [comment]
good

**Reply by snehesht** [reply]
`pip install wheel` fixed it.

**Reply by sidroopdaska** [reply]
Can you share more details for my understanding:
1. OS
2. What virtual env you used

**Reply by GeneralSubhra** [reply]
1.win 11
2.python 3.11
